jobs/config.json

```diff
@@ -11715,7 +11715,8 @@
   },
   "maintenance-pull-scale-janitor": {
     "args": [
-      "--mode=scale"
+      "--mode=scale",
+      "--ratelimit=5"
```

Comment:
we might want to explore higher limits? find the highest that works? /shrug

Reply:
I'll look into how to better to bulk operations over gcloud cli :-\ or if the gcloud client library has a way  to handle this then I'll rewrite janitor in go.